examples/asyncio/audit/audit_examples/log_bulk_async.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+import os
+import time
+
+import pangea.exceptions as pe
+from pangea.asyncio.services import AuditAsync
+from pangea.config import PangeaConfig
+from pangea.services.audit.models import Event
+from pangea.tools import logger_set_pangea_config
+
+token = os.getenv("PANGEA_AUDIT_TOKEN")
+domain = os.getenv("PANGEA_DOMAIN")
+config = PangeaConfig(domain=domain)
+audit = AuditAsync(token, config=config, logger_name="audit")
+logger_set_pangea_config(logger_name=audit.logger.name)
+
+# This example shows how to perform an audit log
+
+
+async def main():
+    print("Logging bulk...")
+
+    event1 = Event(
+        message="Sign up",
+        actor="pangea-sdk",
+    )
+
+    event2 = Event(
+        message="Sign in",
+        actor="pangea-sdk",
+    )
+
+    start = time.time()
+    try:
+        await audit.log_bulk_async(events=[event1, event2], verbose=True)
+    except pe.AcceptedRequestException:
```

Comment:
This is the happy path, I think it shouldn't be an exception

Reply:
Fixed